main.html

```diff
@@ -174,30 +194,31 @@ <h1>CONTACT US</h1>
             <input
               type="email"
               id="mail"
-              placeholder="jeremy@gmail.com"
+              placeholder="Email Address"
               name="email"
               required
             />
           </li>
           <li>
             <textarea
               id="msg"
-              placeholder="Enter text here"
+              placeholder="Write your message here"
               name="message"
               maxlength="500"
               required
             ></textarea>
           </li>
           <li>
-            <div>
-              <button class="submit">See Project</button>
+            <div class="btn1">
+              <button type="submit">Send Message</button>
             </div>
           </li>
         </ul>
       </form>
-    </footer><hr>
+    </footer>
```

Comment:
- [ ] In the contact form, there should be 2 different fields for the first name and last on desktop view as shown in the original design.
- [ ] Instead of "Contact US" the form heading should change to "Let's chat!".

Reply:
@emmanuellekamwa I have made the changes to the logo and replaced with my name and also I have made the changes and included Let's chat! but as u told about two separate fields for first and last name. its mentioned that we don't have to create it. We just have to create one input field as full name for all versions. Kindly look at the link of the requirements and let me know. https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md